dotCMS/src/main/java/com/dotmarketing/osgi/GenericBundleActivator.java

```diff
@@ -814,38 +836,16 @@ protected void unregisterServices ( BundleContext context ) throws Exception {
      */
     protected void unpublishBundleServices () throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)